test/slow/api_spec_slow.js

```diff
@@ -197,15 +197,43 @@ describe(`electron-forge API (with installer=${installer.substr(12)})`, () => {
       const genericTargets = fs.readdirSync(path.resolve(__dirname, '../../src/makers/generic')).map(file => path.parse(file).name);
 
       [].concat(targets).concat(genericTargets).forEach((target) => {
-        describe(`make (with target=${target})`, () => {
-          before(async () => {
-            const packageJSON = await readPackageJSON(dir);
-            packageJSON.config.forge.make_targets[process.platform] = [target];
-            await fs.writeFile(path.resolve(dir, 'package.json'), JSON.stringify(packageJSON));
-          });
-
-          it('successfully makes with default config', async () => {
-            await forge.make({ dir, skipPackage: true });
+        [{
+          format: 'missing',
+          value: undefined,
+          expectFailure: true,
+        }, {
+          format: 'simple-string',
+          value: 'First Last',
+        }, {
+          format: 'full-string',
+          value: 'First Last <first.last@example.com> (http://www.example.com/)',
+        }, {
+          format: 'object',
+          value: {
+            name: 'First Last',
+            email: 'first.last@example.com',
+            url: 'http://www.example.com/',
+          },
+        }].forEach((author) => {
```

Comment:
This will result in a massive increase in build time (12 minutes on the linux machines)

The functionality that turns `author` into a string should be unit tested and this test should be left the same 👍